app/views/layouts/application.html.haml

```diff
@@ -8,5 +8,4 @@
     = stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_pack_tag 'application', 'data-turbolinks-track': 'reload'
   %body
-    .container
-      = yield
+    = yield
```

Comment:
random question, but what does `yield` do? 😄

Reply:
From my own understanding po, `yield` is a layout directive or _block_ in Ruby and Rails - and is widely used to template user interfaces. In addition, `yield` may act as a _wildcard_ in layouts without any arguments passed to it which defaults to rendering context from an existing controller/action since it's also considered as a part of the layout feature of Action View and Action Controller in Rails.

Also, a named `yield` allows you to render content from the `content_for` method which is highly effective when building your dynamic views. 

I've also used its equivalent po before from Laravel Blade. `@yield()`